### PR TITLE
DT-2166 - add trivy image scanning job

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,6 +91,9 @@ workflows:
       - hmpps/gradle_owasp_dependency_check:
           context:
             - hmpps-common-vars
+      - hmpps/trivy_latest_scan:
+          context:
+            - hmpps-common-vars
```
Comment (andrewrlee) [inline .circleci/config.yml:96]: would it make sense parameterising these so it doesn't just go to dps-security-alerts or whatever the default is?

Maybe we could remove some of the defaulting the orb does generally if there isn't a sensible default.

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (mattops) [reply .circleci/config.yml:96]: Yes I did think that, but Kotlin projects dont have a shared slack channel like you have for typescript.  I'll add it in, like you say good to have it there.

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**